ldaptor/config.py

```diff
@@ -32,7 +32,7 @@ def __init__(self,
             self.baseDN = baseDN
         self.serviceLocationOverrides = {}
         if serviceLocationOverrides is not None:
-            for k,v in serviceLocationOverrides.items():
+            for k, v in list(serviceLocationOverrides.items()):
```

Comment:
If `serviceLocationOverrides` is not being modified while iterating, and ldaptor is already using `six`, use `six.iteritems(serviceLocationOverrides)` instead?

Reply:
good point. Or we can leave this as it is in master  ... I was fooled by self.serviceLocationOverrides and thought that it is changed here.

will revert. thanks for looking into this :)